UnitsNet/UnitMath.cs

```diff
@@ -9,64 +10,39 @@ namespace UnitsNet
-            where TUnitType : struct, Enum
-            where TQuantity : IQuantity<TUnitType>
-        {
-            return (TQuantity) Quantity.From(source.Sum(x => x.As(unitType)), unitType);
-        }
-
         /// <summary>
-        ///     Computes the sum of the sequence of <typeparamref name="TQuantity" /> values that are obtained by invoking a
-        ///     transform function on each element of the input sequence.
+        ///     Returns the absolute value of the specified quantity.
         /// </summary>
-        /// <param name="source">A sequence of values that are used to calculate a sum.</param>
-        /// <param name="selector">A transform function to apply to each element.</param>
-        /// <param name="unitType">The desired unit type for the resulting quantity</param>
-        /// <typeparam name="TSource">The type of the elements of source.</typeparam>
-        /// <typeparam name="TQuantity">The type of quantity that is produced by this operation.</typeparam>
-        /// <typeparam name="TUnitType">The type of unit enum.</typeparam>
-        /// <returns>The sum of the projected values, represented in the specified unit type.</returns>
-        /// <exception cref="T:System.ArgumentNullException">
-        ///     <paramref name="source">source</paramref> or <paramref name="selector">selector</paramref> is null.
-        /// </exception>
-        /// <exception cref="ArgumentException">
-        ///     <paramref name="source">source</paramref> contains quantity types different from <paramref name="unitType" />.
+        /// <typeparam name="TQuantity">
+        ///     The type of the quantity, which must implement <see cref="IQuantity" />.
+        /// </typeparam>
+        /// <param name="value">
+        ///     The quantity whose absolute value is to be calculated.
+        /// </param>
+        /// <returns>
+        ///     A quantity of type <typeparamref name="TQuantity" /> representing the absolute value of the input quantity.
+        /// </returns>
+        /// <exception cref="NullReferenceException">
+        ///     Thrown if the input <paramref name="value" /> is <c>null</c>.
         /// </exception>
-        public static TQuantity Sum<TSource, TQuantity, TUnitType>(this IEnumerable<TSource> source, Func<TSource, TQuantity> selector, TUnitType unitType)
-            where TUnitType : struct, Enum
-            where TQuantity : IQuantity<TUnitType>
+        public static TQuantity Abs<TQuantity>(this TQuantity value) where TQuantity : IQuantity
         {
-            return source.Select(selector).Sum(unitType);
+            // TODO see about constraining to IQuantityInstance<TQuantity>
+// #if NET
+//             return TQuantity.Create(QuantityValue.Abs(value.Value), value.UnitKey);
+// #else
+//             return value.QuantityInfo.Create(QuantityValue.Abs(value.Value), value.UnitKey);
+// #endif
+            return (TQuantity)value.QuantityInfo.From(Math.Abs(value.Value), value.UnitKey);
```

Comment:
Eh, one of the commented-out bits has slipped in without me noticing, but the `TODO` is valid (but that's another breaking change).

Personally, I think we should deprecate this method and make it a static method in the `Quantity.cs`: e.g. `var absMass = Quantity.Abs(mass)`.

Alternatively, this could be a static method on the concrete type- e.g. `Mass.Abs(mass)` but that would add bytes to build (we could also have it as a static extension property in `net10`).